content/en/guides/models/app/settings-page/user-settings.md

```diff
@@ -33,16 +33,35 @@ To add a follow badge for the `@weights_biases` account on X, you could add a Ma
 ```
 In an `<img>` tag, you can specify `width`, `height`, or both. If you specify only one of them, the image's proportions are maintained.
 
-## Teams
+## Default team
+If you are a member of more than one team, the **Default team** section allows you to configure the default team to use when a run or a Weave trace does not specify a team. If you are a member of only one team, that team is the default and this section does not appear.
+
+Select a tab to continue.
+
+{{< tabpane >}}
+{{% tab header="Multi-tenant Cloud" %}}
+Next to **Default location to create new projects in**, click thew drop-down, then select your default team.
```

Comment:
I don't think this is rendering properly in the preview
<img width="861" height="201" alt="Screenshot 2025-08-11 at 12 19 18 PM" src="https://github.com/user-attachments/assets/532fce1d-e958-4eff-8ea1-ee97cfa6f28e" />

Reply:
Thanks for noticing this. I often forget the `text=true` on the tabpane. Another thing to note is that the tabpane needs the `<>` which denotes a HTML shortcode and the tab needs the `%%` which denotes a Markdown shortcode. This is an area of Hugo / Docsy I don't fully understand tbh.